tests/test_spark.py

```diff
@@ -59,6 +59,19 @@ def test_task_not_lost():
     utils.check_job_output(driver_task_id, "Pi is roughly 3")
 
 
+@pytest.mark.sanity
+def test_mesos_label_support():
+    driver_task_id = utils.submit_job(app_url=utils.SPARK_EXAMPLES,
+                                      app_args="150",
+                                      args=["--conf spark.cores.max=1",
+                                            "--conf spark.mesos.driver.labels=foo:bar", # pass a test label
+                                            "--class org.apache.spark.examples.SparkPi"])
+
+    driver_task_info = sdk_cmd._get_task_info(driver_task_id)
+    test_label_values = [label['value'] for label in driver_task_info['labels'] if label['key'] == 'foo']
```

Comment:
it would be great to make it a bit more readable. Is it possible to make it look something like that?
```
assert {'foo':'bar'} in driver_task_info['labels']
```

Reply:
changed as suggested